<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PHIEstimatorViewControls</class>
 <widget class="QWidget" name="PHIEstimatorViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>173</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Example View</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_DataSelection">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox::title {
color: rgb(0, 170, 255);
}
</string>
     </property>
     <property name="title">
      <string>Input selection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QmitkDataStorageComboBox" name="m_cbWorkingNodeSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_Segmentation">
        <property name="text">
         <string>Seeds Image</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QmitkDataStorageComboBox" name="m_cbReferenceNodeSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_PatientImage">
        <property name="text">
         <string>Reference Image</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="processImageButton">
     <property name="toolTip">
      <string>Process selected image</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover:!pressed
{
    border-width: 1px;
    border-style: solid;
    border-color: rgb(0, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Run PHi Estimator</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Result">
     <property name="styleSheet">
      <string notr="true">QGroupBox::title {
color: rgb(0, 170, 255);
}
</string>
     </property>
     <property name="title">
      <string>Results</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QLineEdit" name="PhiRatioValue">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="TumorTypeLabel">
        <property name="text">
         <string>Based on this threshold and resulting PHI value this tumor type is</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="PhiRatioLabel">
        <property name="text">
         <string>(Near:Far) Peak Height Ratio</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="PhiLabel">
        <property name="text">
         <string>PHI</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="PhiValue">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="NearROIVoxelsUsedLabel">
        <property name="text">
         <string>Near ROI voxels used</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="PHIThresholdValue">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="FarROIVoxelsUsedValue">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="NearROIVoxelsUsedValue">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="FarROIVoxelsUsedLabel">
        <property name="text">
         <string>Far ROI voxels used</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="PHIThresholdLabel">
        <property name="text">
         <string>PHI Threshold (based on 142 UPenn brain tumor scans)</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="TumorTypeValue">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">QmitkDataStorageComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../workspace/MITK/CaPTk2.0/Plugins/upenn.cbica.captk.interactivesegmentation/resources/captkinteractivesegmentation.qrc"/>
 </resources>
 <connections/>
</ui>
